cli/client.go

```diff
@@ -110,6 +110,73 @@ func (c *viamClient) listOrganizationsAction(cCtx *cli.Context) error {
 	return nil
 }
 
+// OrganizationsSupportEmailSetAction corresponds to `organizations support-email set`.
+func OrganizationsSupportEmailSetAction(cCtx *cli.Context) error {
+	c, err := newViamClient(cCtx)
+	if err != nil {
+		return err
+	}
+
+	orgID := cCtx.String(generalFlagOrgID)
+	if orgID == "" {
+		return errors.New("cannot set support email without an organization ID")
+	}
+
+	supportEmail := cCtx.String(organizationFlagSupportEmail)
+	if supportEmail == "" {
+		return errors.New("cannot set support email to an empty string")
+	}
```

Comment:
idk what our convention is for this in our cli code, but if we check this server side, do w need this validation here? doing it in both places makes it easy to get out of sync